instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb

```diff
@@ -37,6 +37,9 @@ def call(context) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metri
             else
               super
             end.tap do |response|
+              span.set_attribute(OpenTelemetry::SemanticConventions::Trace::HTTP_STATUS_CODE,
+                                 context.http_response.status_code)
```

Comment:
Is it guaranteed that `context.http_response` is present in all services and operations?
Should we consider the use of [safe navigation operator](https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) so this code will no throw if the value is `Nil` for some reason?

Reply:
I now read the `successful?` implementation you attached and saw that it's safe to use it 👍